osidb/serializer.py

```diff
@@ -1291,7 +1291,7 @@ def get_affects(self, obj):
                     trackers__external_system_id__in=tracker_ids.split(",")
                 )
 
-        serializer = AffectSerializer(instance=affects, many=True, context=context)
+        serializer = AffectSerializer(instance=affects, many=True, read_only=True, context=context)
```

Comment:
```
        serializer = AffectSerializer(
            instance=affects, many=True, read_only=True, context=context
        )
```
to make black hapy :)

Reply:
thx, I was getting my local env setup ... all good now ;)